App/Controllers/SolicitacaoController.php

```diff
@@ -281,6 +281,13 @@ public function processarAction()
         (new SolicitacaoModel())->processStatus($id, $status, $action);
 
         header('location: ' . $this->view->controller);

```

Comment:
Por favor, envolva essa configuração de header em uma estrutura `else`.
Não faz sentido setar o header duas vezes. (nos casos em que a condição no `if` da linha 285 for satisfeita)

```php
if ($status == 'APROVADO' && $action == 'PROXIMO') {
    ...
} else {
    header('location: ' . $this->view->controller);
}
```

Reply:
Concordo @br-monteiro, não faz sentido mesmo. Obrigado :)